### DISCUSSION TITLE
如何在win10获取公共ipv6地址 dufault和public只能获得临时的

### DISCUSSION THREAD

**Discussion by wkumo** [NewFuture/DDNS#251] [Q&A] [answered]

**Comment by lipaopao000** [comment]
哈哈，我有同样问题。Windows平台无法正则匹配

**Comment by NewFuture** [comment]
@lipaopao000 正则有啥问题？

**Comment by szl0834** [comment]
> @lipaopao000 正则有啥问题？

正则只能按照前缀或者后缀来筛选啊，遇到前后缀都不固定的情况下就没办法指定获得非临时ip了

**Comment by irrocu** [comment]
这个问题是不是不能解决呢，DDNS插件我用在Win10下，不能获取首选ipv6地址进行更新，而是选择临时ipv6地址进行同步更新到阿里云。

**Comment by imldy** [comment]
+1，这个怎么处理呢

**Comment by NewFuture** [comment]
可以有选网卡的，或者自定义命令来获取

**Reply by werfei** [reply, marked as answer]
可以禁用临时ipv6地址
`
netsh interface ipv6 set privacy state=disable
`
